parser.y

```diff
@@ -1157,6 +1157,7 @@ import (
 	OptCharset                             "Optional Character setting"
 	OptCollate                             "Optional Collate setting"
 	IgnoreLines                            "Ignore num(int) lines"
+	Int64Num                               "A number that can be converted to signed 64 bits integer"
```

Comment:
what's the difference with `num`?

Reply:
It checks if it is ok to cast to `int64`.